Mage/src/main/java/mage/game/stack/Spell.java

```diff
@@ -1017,6 +1017,11 @@ public boolean addCounters(Counter counter, UUID playerAddingCounters, Ability s
         return card.addCounters(counter, playerAddingCounters, source, game, appliedEffects, isEffect, maxCounters);
     }
 
+    @Override
+    public boolean addCounters(Counter counter, UUID playerAddingCounters, Ability source, Game game, List<UUID> appliedEffects, boolean isEffect, int maxCounters, boolean informPlayers) {
+        return card.addCounters(counter, playerAddingCounters, source, game, appliedEffects, isEffect, maxCounters, informPlayers);
```

Comment:
Is this going to work for copied spells? Do they have a non-null card value?

Reply:
I'm not sure where this is called from honestly.  The "find usages" in IntelliJ is combining this with Card since Spell implements Card.  It looks like this is the way it's been for a long time though.  I just added another method for the informPlayers flag.

Reply:
Lightning Storm is an example of a card that can have counters put on it while it's on the stack (might be the only one)

Reply:
> Lightning Storm

a) How do you know that the counters should go on the card rather than the spell on the stack?
b) I guess we can write a test for casting Lightning Storm and copying it.

Reply:
What do you want to test? In Magic, copying an object (card, spell or permanent) doesn't copy any counters on it.

Reply:
> What do you want to test? In Magic, copying an object (card, spell or permanent) doesn't copy any counters on it.

I was asking if the copy of a spell has a non-null `card` instance variable. I don't know how we handle copying for that.